### ISSUE TITLE
V0.2.5 Not running under the Guest Account

### ISSUE THREAD

**Issue by red5coder** [UPBGE/upbge#1483]
Tested on macOS 10.15 and 11.3.1. We cannot run UPBGE under the built in macOS guest account. Local admins and non admins are fine.

When under the guest account, it complains that it does not have the correct permissions to run.

**Comment by lordloki** [comment]
Have you check to modify the permissions of the Blender.app or Blenderplayer.app once installed?
Finder->Applications->Blender->right click->Information->rights->modify permissions for everyone
![image](https://user-images.githubusercontent.com/1015605/118919833-90e0f800-b935-11eb-8b6c-400753d858e7.png)

**Comment by red5coder** [comment]
Had some success. It wasn't the permissions. We gave everyone read/write etc.

Because they had the other version of Blender installed we installed this version in a sub folder in /Applications.

This was causing the issue. Even if we gave that directory full read/write.

We renamed the app so it did not clash with the other version and moved it to /Applications.

It now works so I think we can close this.

Thanks for your help
